_data/simple-icons.json

```
@@ -5472,6 +5472,11 @@
             "hex": "003A9B",
             "source": "https://indeed.design/resources"
         },
+        {
+            "title": "Inertia",
+            "hex": "FFFFFF",
```

```suggestion
            "hex": "000000",
```

Let's use black as its primary color.